receiver/cloudfoundryreceiver/README.md

```diff
@@ -0,0 +1,90 @@
+UAA user must have the `client_credentials` and `refresh_token` authorized grant types, and `logs.admin` authority.
+
+The following is an example sequence of commands to create the UAA user using the `uaac` command line utility:
+
+* `uaac target https://uaa.<cf-system-domain>`
+* `uaac token client get identity -s <identity-user-secret>`
+* `uaac client add <uaa_username> --name opentelemetry --secret <uaa_password> --authorized_grant_types client_credentials,refresh_token --authorities logs.admin`
+
+The `<uaa_username>` and `<uaa_password>` above can be set to anything as long as they match the values provided to the
+receiver configuration. The admin account (which is `identity` here) which has the permissions to create new clients may
+have a different name on different setups. The value of `--name` is not used for receiver configuration.
+
+## Configuration
+
+The receiver takes the following configuration options:
+
+| Field | Default | Description |
+| --- | --- | --- |
+| `rlp_gateway_url` | required | URL of the RLP gateway, typically `https://log-stream.<cf-system-domain>` |
+| `rlp_gateway_skip_tls_verify` | `false` | whether to skip TLS verify for the RLP Gateway endpoint |
+| `rlp_gateway_shard_id` | `opentelemetry` | metrics are load balanced among receivers that use the same shard ID, therefore this must only be set if there are multiple receivers which must both receive all the metrics instead of them being balanced between them |
+| `uaa_url` | required | URL of the UAA provider, typically `https://uaa.<cf-system-domain>` |
+| `uaa_skip_tls_verify` | `false` | whether to skip TLS verify for the UAA endpoint |
+| `uaa_username` | required | name of the UAA user (required grant types/authorities described above) |
+| `uaa_password` | required | password of the UAA user |
+| `http_timeout` | `10s` | HTTP socket timeout used for RLP Gateway |
+
+Example:
+
+```yaml
+receivers:
+  cloudfoundry:
+    rlp_gateway_url: "https://log-stream.sys.example.internal"
+    rlp_gateway_skip_tls_verify: false
+    rlp_gateway_shard_id: "opentelemetry"
+    uaa_url: "https://uaa.sys.example.internal"
+    uaa_skip_tls_verify: false
+    uaa_username: "otelclient"
+    uaa_password: "changeit"
+    http_timeout: "20s"
+```
+
+The full list of settings exposed for this receiver are documented [here](./config.go)
+with detailed sample configurations [here](./testdata/config.yaml).
+
+## Metrics
+
+Reported metrics are grouped under an instrumentation library named `otelcol/cloudfoundry`. Metric names are as
+specified by [Cloud Foundry metrics documentation](https://docs.cloudfoundry.org/running/all_metrics.html), but the
+origin name is prepended to the metric name with `.` separator. All metrics either of type `Gauge` or `Sum`.
+
+### Attributes
+
+All the metrics have the following attributes:
+* `origin` - origin name as documented by Cloud Foundry
```

Comment:
Is origin both an attribute and also prepended to the metric name? (I don't know if this is desirable or no, just double checking that the docs are correct).